tcms/telemetry/templates/telemetry/testing/execution-trends.html

```diff
@@ -76,6 +76,21 @@
             </div>
         </form>
 
+
+        <div class="passing-rate-summary">
+            <div class="progress-description" style="display: flex; justify-content: space-between; max-width:100%">
+                <div>
+                    <span class="pficon pficon-ok"></span><strong>Passed</strong>
+                </div>
+                <div>
+                    <span class="pficon pficon-error-circle-o"></span><strong>Not passed</strong>
+                </div>
+            </div>
+            <div class="progress">
+                <div class="progress-bar progress-bar-success" role="progressbar"></div>
+                <div class="progress-bar progress-bar-danger" role="progressbar"></div>
+            </div>
+        </div>
```

Comment:
If you look at the proposed picture in the blog post, on the right hand side there is a box showing total numbers. You can add TOTAL above the progress bar, then Postive, Neutral, Negative numbers + color codes under the status bar like a legend markers and this page will be ready to go.